test/create_load_test.js

```diff
@@ -7,35 +7,13 @@ var Promise = require('promise');
 var stats   = require("taskcluster-lib-stats");
 var debug   = require('debug')('test:entity:create_load');
 
-suite("Entity (create/load)", function() {
-  var ItemV1;
-  test("ItemV1 = Entity.configure", function() {
-    ItemV1 = subject.configure({
-      version:          1,
-      partitionKey:     subject.keys.StringKey('id'),
-      rowKey:           subject.keys.StringKey('name'),
-      properties: {
-        id:             subject.types.String,
-        name:           subject.types.String,
-        count:          subject.types.Number
-      }
-    });
-  });
-
-  var Item;
-  test("Item = ItemV1.setup", function() {
-    Item = ItemV1.setup({
-      credentials:  helper.cfg.azure,
-      table:        helper.cfg.tableName
-    });
-  });
+var makeTests = function(Item, Item2, drain) {
```

Comment:
I'll try to play with it a bit -- I could potentially have a helper that calls suite() twice